taskqueue/gcloud/aio/taskqueue/taskmanager.py

```diff
@@ -59,7 +59,7 @@ def start(self, loop=None):
 
     @staticmethod
     def autorenew(event, headers, task, lease_seconds):
-        url = f'{API_ROOT}/{task["name"]}:renewLease'
+        url = f'{API_ROOT}/v2beta2/{task["name"]}:renewLease'
```

Comment:
Could we make the base api root variables static so we could define this as `PullQueue.base_api_root`? That'd be nicer than duplicating the api versions.

Reply:
will do!